moveit_kinematics/CMakeLists.txt

```diff
@@ -21,9 +21,6 @@ find_package(catkin REQUIRED COMPONENTS
   tf_conversions
 )
 
-find_package(trac_ik_kinematics_plugin)
```

Comment:
These are only required for `cached_ik_kinematics_plugin`.

Reply:
I usually prefer to have all `find_package` calls for a project in a single file even if they are only used in some subfolder(s). This avoids confusion when new users look for options/dependencies in the cmake structure.
In this case I don't have a strong opinion though.